src/Evolution/Executables/CurvedScalarWave/EvolveCurvedScalarWave.hpp

```diff
@@ -110,6 +120,21 @@ class er;
 }  // namespace PUP
 /// \endcond
 
+// We want to integrate psi^2 on a spherical shell
+struct PsiSquared : db::SimpleTag {
```

Comment:
Would it be useful to stick this into `Evolution/Systems/CurvedScalarWave/PsiSquared.hpp`? Then it's easily tested too